src/EventExport/Command/ExportEventsAsPDF.php

```diff
@@ -74,6 +78,19 @@ private function setAddress(EmailAddress $address): void
         $this->address = $address;
     }
 
+    public function withSortOrder(SortOrder $sortOrder): ExportEventsAsPDF
+    {
+        $exportEvents = clone $this;
+        $exportEvents->setSortOrder($sortOrder);
+
+        return $exportEvents;
+    }
+
+    private function setSortOrder(SortOrder $sortOrder): void
```

Comment:
Same here, no real use for a private setter

Reply:
[Replace sortOrder setters by property assignment](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/0029539335c3896a4b4030ccb4eb3d767e988d1c)